src/pages/version/BuildVariantCard/VariantTaskGroup/VariantTaskGroup.test.tsx

```diff
@@ -0,0 +1,317 @@
+      expect(failedBadge).toHaveAttribute("aria-selected", "false");
+    });
+    it("none should be active if no matching variant is selected", () => {
+      render(<Component />, {
+        wrapper: Wrapper,
+        route: `/version/1?statuses=${
+          TaskStatus.Succeeded
+        }&variant=${applyStrictRegex("some_other_variant")}`,
+        path: "/version/:id",
+      });
+      expect(screen.getByText("Some Variant")).toBeDefined();
+      expect(screen.queryAllByDataCy("grouped-task-status-badge")).toHaveLength(
+        2
+      );
+      screen.queryAllByDataCy("grouped-task-status-badge").forEach((badge) => {
+        expect(badge).toHaveAttribute("aria-selected", "false");
+      });
+    });
+  });
+
+  describe("variantTaskGroup links", () => {
+    describe("title", () => {
+      it("should link to the variant filter if there is no variant selected", () => {
+        render(<Component />, {
+          wrapper: Wrapper,
+          route: `/version/1`,
+          path: "/version/:id",
+        });
+        const variantLink = screen.queryByDataCy("build-variant-display-name");
+        expect(variantLink).toBeDefined();
+        expect(variantLink).toHaveTextContent("Some Variant");
+        expect(variantLink).toHaveAttribute(
+          "href",
+          getVersionRoute("1", {
+            variant: applyStrictRegex("some_variant"),
+            page: 0,
+          })
+        );
+      });
+      it("should link to the variant filter if a different variant is selected", () => {
+        render(<Component />, {
+          wrapper: Wrapper,
+          route: `/version/1?variant=${applyStrictRegex("some_other_variant")}`,
+          path: "/version/:id",
+        });
+        const variantLink = screen.queryByDataCy("build-variant-display-name");
+        expect(variantLink).toBeDefined();
+        expect(variantLink).toHaveTextContent("Some Variant");
+        expect(variantLink).toHaveAttribute(
+          "href",
+          getVersionRoute("1", {
+            variant: applyStrictRegex("some_variant"),
+            page: 0,
+          })
+        );
+      });
+      it("should remove the variant filter if the same variant is selected", () => {
+        render(<Component />, {
+          wrapper: Wrapper,
+          route: `/version/1?variant=${applyStrictRegex("some_variant")}`,
+          path: "/version/:id",
+        });
+        const variantLink = screen.queryByDataCy("build-variant-display-name");
+        expect(variantLink).toBeDefined();
+        expect(variantLink).toHaveTextContent("Some Variant");
+        expect(variantLink).toHaveAttribute(
+          "href",
+          getVersionRoute("1", {
+            page: 0,
+          })
+        );
+      });
+    });
+    describe("status badges", () => {
+      it("should link to the variant and status filter if there is no variant selected", () => {
+        render(<Component />, {
+          wrapper: Wrapper,
+          route: `/version/1`,
```

Comment:
nit. Use quotes (there's some other places in the file where this applies too)
```suggestion
          route: "/version/1",
```